src/certifier.mak

```diff
@@ -1,7 +1,9 @@
 #    
 #    File: certifier.mak
 
-ENABLE_SEV=1
+ifndef NO_ENABLE_SEV
+    ENABLE_SEV=1
```

Comment:
See [the reasons](https://github.com/vmware-research/certifier-framework-for-confidential-computing/issues/215#issuecomment-1783620655) explaining why this was introduced. 

This NO_ENABLE_SEV is OFF by default. It's only invoked currently by one `make` invocation in `test.sh`, so none of the other usages that build a shared lib will be affected.